process/tech-due-diligence-guidelines.md

```diff
@@ -0,0 +1,103 @@
+# Technical Due Diligence Guidelines
+
+This page provides guidelines to those leading or contributing to technical due diligence exercises performed by or on behalf of the Technical Oversight Committee of the CNCF.
+
+## Introduction
+
+Part of the evaluation process in deciding upon initial or continued
+inclusion of projects into the CNCF is a Technical Due Diligence
+('Tech DD') exercise.  Ultimately the voting members of the TOC will,
+on the basis of this and other information, vote for or against the
+inclusion of each project at the relevant time.
+
+## Leading a Technical Due Diligence
+
+### Primary Goals
+
+To enable the voting TOC members to cast an informed vote about a
+project, it is crucial that each member is able to form their own
+opinion as to whether and to what extent the project meets the agreed
+upon [criteria](https://www.cncf.io/projects/graduation-criteria/) for
+inception, incubation or graduation. As the leader of a DD, your job
+is to make sure that they have whatever information they need,
+succinctly and readily available, to form that opinion.
+
+As a secondary goal, it is in the interests of the broader CNCF
+ecosystem that there exists some reasonable degree of consensus across
+the community regarding the inclusion or otherwise of projects at the
+various maturity levels.  Making sure that the relevant information is
+available, and any disagreement or misunderstanding as to it's
+validity are ideally resolved, helps to foster this consensus.
+
+### Where to start
```

Comment:
What are we thinking with regard to the appointing a designated tech DD team or committee to be accountable for the tech DD along with the TOC sponsor? Or would we rather this be organic?

Reply:
so far its been mostly self-nominations along with input from @monadic

Reply:
organic sgtm; in the end the proposal / dd have to win votes

Reply:
I think it's practically useful to have one or more explicit leads for the due diligence, whose job it is to steer the process to completion.  In the absence of that, the process tends to drag on unnecessarily.  Thoughts?

Reply:
what if the project's TOC sponsor picks one or more contributors to help with the DD. I believe that's what @monadic did for jaeger and envoy. other community feedback is obviously welcome.

Reply:
If the TOC wants to, I think it would be fine for the TOC sponsor to pick a TOC contributor to lead the due diligence. But part of the TOC contributor idea was that many of CNCF member companies have deep wells of expertise on a variety of subjects.

I would always want to encourage any TOC contributor who was willing (and especially if they had internal colleagues with experience using that project) to share as much information as possible.

Reply:
OK, agreed. I think I have all this captured appropriately in the doc.  Let me know if you disagree.

Reply:
Should it be up to the CNCF Sigs to determine the criteria for evaluating each of these?

Reply:
CNCF has 4 working groups but the TOC has never delegated authority to them. Anything they propose must eventually be adopted by the TOC.